package.json

```
@@ -32,6 +32,7 @@
   ],
   "devDependencies": {
     "@types/node": "17.0.8",
+    "deno2node": "^1.1.0",
     "prettier": "2.5.1",
     "typescript": "4.5.4"
```

`typescript` is redundant.
```suggestion
    "prettier": "2.5.1"
```

---

Removed.